source/blog/2014-02-04-ember-1-10-0-released.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Ember.js 1.10.0 and 1.11 Beta Released
+author: Matthew Beale
+tags: Releases, Recent Posts
+---
+
+We are pleased to announce the release of both Ember.js 1.10.0 and the
+first beta in the 1.11 series. This comes as the tenth cycle of our
+release process that began just after 1.0 was released.
+
+The 1.10 release represents the effort of at least 50 contributors
+across over 553 commits.
+
+## The HTMLBars Templating Engine
+
+Throughout 2014 the Ember.js community has poured its shared effort into
+a new templating solution. This new library, dubbed HTMLBars, makes available
+new feature and performance improvements impossible to support with Handlebars
+(the previous library). We are delighted to announce the inclusion of
+the HTMLBars templating engine in Ember.js 1.10.
+
+HTMLBars will enable the implementation of several APIs described in
+[The Road to Ember 2.0](https://github.com/emberjs/rfcs/pull/15), and contributes
+to our continued improvement of rendering performance. Additionally,
+it will allows us to continue supporting all Ember 1.x template syntax.
+
+This long-term effort would not have succeeded without the continued
+contributions of the Ember community and core team. In no particular
+order and with apologies to anyone we've missed we would like to thank
+[@krisseldon](http://twitter.com/krisseldon'), [@wycats](http://twitter.com/wycats),
+[@\_mmun](http://twitter.com/_mmun), [@johanneswuerbach](http://twitter.com/nerd20),
+[@mixonic](http://twitter.com/mixonic), [@ebryn](http://twitter.com/ebryn),
+[@salzhrani](https://twitter.com/salzhrani), [@rwjblue](http://twitter.com/rwjblue),
+[@tomdale](http://twitter.com/tomdale), [oneeman](http://github.com/oneeman),
+[@kpdecker](http://twitter.com/kpdecker) and the many alpha and beta
+testers who provided feedback and bug reports. You are the stuff "stability without
+stagnation" is made of.
+
+## New Features in Ember.js 1.10
+
+#### HTMLBars 0.9.0
+
+Ember.js 1.10 is dependent on HTMLBars 0.9.0. To ease this and later template
+library version changes, Ember's template compiler will be packaged as a
+part of the release files. For example, a JSBin in Ember 1.9 would need to be
+updated from:
+
+```
+<script src="http://builds.handlebarsjs.com.s3.amazonaws.com/handlebars-v2.0.0.js"></script>
+<script src="http://builds.emberjs.com/tags/v1.9.1/ember.js"></script>
+```
+
+to
+
+```
+<script src="http://builds.emberjs.com/tags/v1.10.0/ember-template-compiler.js"></script>
+<script src="http://builds.emberjs.com/tags/v1.10.0/ember.debug.js"></script>
+```
+
+The `ember-template-compiler.js` is only required for template compilation. The
+runtime dependencies for HTMLBars are built into the framework file.
+
+Ember-CLI users should update to a version at or after 0.1.12, then remove
+their application's Handlebars dependency. Ember App-Kit users should upgrade
+to Ember-CLI
+
+Ember-Rail users should upgrade to version X.XX, then remove their Handlebars
+dependency.
```

Comment:
@tricknotes @rwjblue I need some guidance here. Is this the proper path for Ember-Rails users?

In fact are the above instructions generally sane?